srcpkgs/nautilus/template

```diff
@@ -17,7 +17,8 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=37ce2c16a610c589dcc7660f9092446465568e38e29bce6ed8c24f2e8e0077f3
+checksum=bb06e226b965f479a0b8668c19c8e1762b9d8f039af6817961cc7a8f09e6f8a7
+patch_args="-Np1"
```

Comment:
I guess you forget to include `patches` directory.

As of it's now, nautilus requires pango>=1.44 for
`pango_attr_insert_hyphens_new`